content/realtime/inband-occupancy.textile

```diff
@@ -0,0 +1,124 @@
+---
+title: Inband channel occupancy events
+meta_description: "Inband occupancy events are events on a user-defined channel that contains occupancy data for the channel."
```

Comment:
Again, is "user-defined channel" now a defined term?

Reply:
No, I shouldn't be introducing terms on the fly. Changed.